crates/debugger/src/lib.rs

```diff
@@ -924,8 +969,11 @@ Line::from(Span::styled("[t]: stack labels | [m]: memory decoding | [shift + j/k
                     .map(|(j, byte)| {
                         Span::styled(
                             format!("{byte:02x} "),
-                            if let (Some(w), Some(color)) = (word, color) {
-                                if (i == w / 32 && j >= w % 32) || (i == w / 32 + 1 && j < w % 32) {
+                            if let (Some(w), Some(size), Some(color)) = (word, size, color) {
+                                if (i == w / 32 && j >= w % 32) ||
+                                    (i > w / 32 && i < (w + size - 1) / 32) ||
+                                    (i == (w + size - 1) / 32 && j <= (w + size - 1) % 32)
```

Comment:
Can we add some docs here too? actually not super clear on what this does without comments.

Reply:
Added comments. Now should be clearer.